### PR TITLE
Include rmc fix and use pigz

### PR DESCRIPTION
**Pull request by mkumatag**
Changes:
- included refresh_rmc_and_interface
- Using pigz if present for faster compress
- Formatting

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/15#" title="Author self-approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: scripts/images/convert_qcow2_ova.py
Comment (bpradipt) [inline scripts/images/convert_qcow2_ova.py:251]: Is the error message correct ?

Comment (mkumatag) [reply scripts/images/convert_qcow2_ova.py:251]: copy-paste error, good find :P

Comment (mkumatag) [reply scripts/images/convert_qcow2_ova.py]: fixed it, ptal latest code.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/15#pullrequestreview-494561584" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/15#" title="Author self-approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [bpradipt,mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
